messaging-modules/automq/src/main/java/com/baeldung/automq/UserOnboardingInitiatedListener.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.automq;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.KafkaListener;
+
+@Configuration
+public class UserOnboardingInitiatedListener {
+
+    private static final Logger log = LoggerFactory.getLogger(UserOnboardingInitiatedListener.class);
+
+    @KafkaListener(topics = { "${com.baeldung.topic.onboarding-initiated}" }, groupId = "user-service")
```

Comment:
Is it possible to use the shorthand for a single element array here?
```suggestion
    @KafkaListener(topics = "${com.baeldung.topic.onboarding-initiated}", groupId = "user-service")
```

Reply:
Yes, review incorporated.